packages/common/src/plugins/requestValidation/useCheckGraphQLQueryParam.ts

```diff
@@ -0,0 +1,25 @@
+import { createGraphQLError } from '@graphql-tools/utils'
+import { Plugin } from '../types'
+
+export function useCheckGraphQLQueryParam(): Plugin {
+  return {
+    onRequestParse() {
+      return {
+        onRequestParseDone({ params }) {
+          if (params.query == null) {
+            throw createGraphQLError('Must provide query string.', {
+              extensions: {
+                http: {
+                  status: 400,
+                  headers: {
+                    Allow: 'GET, POST',
+                  },
+                },
+              },
+            })
+          }
+        },
+      }
+    },
+  }
+}
```

Comment:
I think in order to also make this logic available we should export the wrapped functionality as standalone functions? Not something we need to do here, but should do in general, as then people can also use these functions over helix, without the graphql-yoga class/object/createServer 🤔

Reply:
I agree. That's one of the points of these changes actually (to split "spec" validation logic into individual functions/plugins)